src/internal/connector/onedrive/collection.go

```diff
@@ -273,6 +273,7 @@ func (oc *Collection) populateItems(ctx context.Context) {
 
 	errUpdater := func(id string, err error) {
 		m.Lock()
+		// TODO: Label(fault.LabelForceNoBackupCreation)
```

Comment:
this needs implemented?

Reply:
We're not passing in errs at this point.  That gets implemented in... two PRs from now?

Reply:
doesn't not having some check for onedrive make it unsafe though? I know we don't have incrementals for onedrive yet, but we're already persisting enough metadata for it to technically use a previous backup as a base

Reply:
It is unsafe.  But no more unsafe than we already are with a lack of error tracking in the first place.  We don't have a mechanism for review here until fault gets put into place.  In the meantime, GC support will mark the error as a failure, which should cascade into an early exit all the same.

Reply:
>But no more unsafe than we already are with a lack of error tracking in the first place.

We're fail-fast right now so errors don't cause a backup model to be created avoiding the issue

>In the meantime, GC support will mark the error as a failure, which should cascade into an early exit all the same.

Where is this happening? Is GC supposed to be adding a label for that? Or are we assuming that there will be some error in the fault.Errors that causes the failure? If we switch from fail-fast will GC still be generating that failure?

Reply:
> Where is this happening?

As per the state of this PR: 
* error aggregation with multierrs: https://github.com/alcionai/corso/blob/main/src/internal/connector/onedrive/collection.go#L374
* error reporting in statusUpdater: https://github.com/alcionai/corso/blob/main/src/internal/connector/onedrive/collection.go#L460
* error return on non-zero gc.status errors: https://github.com/alcionai/corso/blob/main/src/internal/operations/backup.go#L279
* readErr exits before backing up the model: https://github.com/alcionai/corso/blob/main/src/internal/operations/backup.go#L625

Reply:
mkay, thanks for the references. Can we add a TODO or a comment to the final part to ensure we don't remove that prior to getting OneDrive fault handling setup?

Also, do we have some way to test this is actually doing what we think it should be if there's a failure? Would be really good to see it in action prior to merging

Reply:
Done.  And no testing that I can easily manufacture.  We're kinda stuck here between complete e2e (no way to force targeted failures) and needing to be able to mock every layer (which we don't have coverage for yet).  I could hack something up for a manual show, if wanted.  But that's not exactly resilient testing.

Reply:
if something manual isn't super difficult I think that would be worth it to ensure things are at least working on some call path

Reply:
@ashmrtn 

Added a forced `et.Add(clues.New("forced failure").Label(fault.LabelForceNoBackupCreation))`.  

```
17:47:00.47	INFO	completed backup; conditional error forcing exit without model persistence	{"error": "forced failure", "errorVerbose": "forced failure\n\t/Users/keepers/go/src/github.com/alcionai/corso/src/internal/connector/exchange/api/mail.go:219", "results": {"bytesRead":930866,"bytesUploaded":949300,"itemsRead":41,"itemsWritten":22,"resourceOwners":1,"startedAt":"2023-02-21T17:46:19.23712-07:00","completedAt":"2023-02-21T17:47:00.475452-07:00","backupID":"696b60f3-6b16-4d33-99c2-5f0f2b950dc3"}}

No backups available
Error: 1 error occurred:
	* Failed to run Exchange backup for user Rfinders@8qzvrj.onmicrosoft.com: forced backup: forced failure
```

Reply:
excellent thanks!

Reply:
You're very welcome!